ensembl/htdocs/info/genome/compara/species_trees.html

```diff
@@ -17,19 +17,72 @@ <h1 id="speciestrees">Species trees</h1>
+
+    Briefly, the tree is computed as follows:
+
+    <ol>
+    <li>The species-tree pipeline begins with a lightweight annotation of the input genomes. This is achieved by aligning a BUSCO <a href="#fn1">[1]</a> protein set to the genomic sequences using miniprot <a href="#fn2">[2]</a> (version 0.12-r237). From these alignments, the corresponding coding sequences (CDS) are extracted.</li>
+
+    <li>Next, the CDS for each gene are aligned across all species to generate codon alignments. This multiple sequence alignment is primarily performed using PAGAN <a href="#fn3">[3]</a> (version v.0.61), with PRANK <a href="#fn4">[4]</a> (version v.170427) used as a fallback option if the former cannot be calculated.</li>
+
+    <li>From the translations of these individual codon alignments, a phylogenetic tree is inferred for each gene using IQ-TREE <a href="#fn5">[5]</a> (2.2.0.3 COVID-edition). These individual gene trees are then reconciled by Astral <a href="#fn6">[6]</a> (version 5.7.1) to generate a consensus species tree.</li>
+
+    <li>This consensus species tree is then used as a starting topology for a refinement process. A supermatrix is created by concatenating the protein alignments of all genes and it is trimmed by using TrimAl <a href="#fn7">[7]</a> (version v1.4.rev15). Then IQ-TREE is used to performs a new, full tree search on this supermatrix to find the optimal species tree topology, using the Astral tree as the starting topology. This step refines both the topology and branch lengths of the final tree.</li>
+
+    <li>Finally, branch lengths for this definitive topology are also computed from the fourfold degenerate sites of the concatenated codon alignment (used in Cactus <a href="#fn8">[8]</a> multiple genome alignments).</li>
+    </ol>
+
     </li>
 </ol>
 
+<h2 id="notes">Notes and References</h2>
+
+<ol>
+
+<li id="fn1">
+Manni M, Berkeley MR, Seppey M, Simão FA, Zdobnov EM. BUSCO Update: Novel and Streamlined Workflows along with Broader and Deeper Phylogenetic Coverage for Scoring of Eukaryotic, Prokaryotic, and Viral Genomes.  <a href="https://europepmc.org/article/MED/34320186">Molecular Biology and Evolution, Volume 38, Issue 10, October 2021, Pages 4647–4654</a>.
+</li>
+
+<li id="fn2">
+Li H. Protein-to-genome alignment with miniprot. <a href="https://europepmc.org/article/MED/36648328">Bioinformatics. 2023 Jan;39(1):btad014</a>.
+</li>
+
+<li id="fn3">
+Löytynoja A, Vilella AJ, Goldman N. Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm. <a href="https://europepmc.org/article/MED/22531217">Bioinformatics. 2012 Jul;28(13):1684-1691</a>.
+</li>
+
+<li id="fn4">
+Löytynoja A, Vilella AJ, Goldman N. Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm. <a href="https://europepmc.org/article/MED/22531217">Bioinformatics. 2012 Jul;28(13):1684-1691</a>.
+</li>
+
+<li id="fn5">
```

Comment:
In conjunction with the deduplication of Löytynoja et al. (2012):
```suggestion
<li id="fn4">
```